src/test/java/org/cactoos/io/TempFileTest.java

```diff
@@ -70,19 +70,19 @@ public void createFileInCustomPath() throws Exception {
                         }
                     )
                 )
-            );
+            ).affirm();
         }
     }
 
     @Test
     public void deleteFile() throws Exception {
         final TempFile file = new TempFile();
         file.close();
-        MatcherAssert.assertThat(
+        new Assertion<>(
             "Cannot delete file on close",
             Files.exists(file.value()),
             Matchers.is(false)
```

Comment:
@marceloamadeu can you change that to `new IsNot<>(IsTrue())` or `new IsEqual<>(false)` ?

Reply:
@victornoel Done ... please check.